modules/feincms-pages/scripts/init.js

```diff
@@ -7,8 +7,7 @@
         'feincms-pages.page-group',
         'feincms-pages.load-page',
         'feincms-pages.constants',
-        'feincms-pages.routes',
-        'feincms-pages.templates'
+        'feincms-pages.routes'
```

Comment:
I found two that have an `init.js` depending on standalone sub-modules, including templates, and both register `mainmodule.templates` as a dependency in the `init.js`. See this search that includes private repos: https://github.com/search?q=org%3Aincuna+standalone+language%3AJavaScript&ref=searchresults&type=Code&utf8=%E2%9C%93 (it searches for the `ngtemplates` option `standalone`, as documented here: https://www.npmjs.com/package/grunt-angular-templates#standalone)